client/src/app/pages/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="dashboard">
+  <div class="tasks-section">
+    <app-controls-in-progress
```

Comment:
Again about names. Why it called controls and in progress?

Reply:
Because those two top panels, _controls-in-progress_ and _controls-completed_, are not the same in that _controls-in-progress_ has a _add task_ button while  _controls-completed_ does not. This is why I had to create two different panels and name them differently. They are named controls because in the future a lot of buttons like sorting and filtering may be added to those panels